api/src/api/controllers/brla.controller.ts

```diff
@@ -367,42 +423,54 @@ export const validatePixKey = async (
 
     res.status(200).json({ valid: true });
   } catch (error) {
-    handleApiError(error, res, 'triggerOnramp');
+    handleApiError(error, res, 'validatePixKey');
   }
 };
 
-// DEPRECATED
+
 /**
- * Trigger an onramp operation
+ * Creates a request for KYC level 2
  *
- * Given the confirmation of payment from the user, it triggers an onramp operation.
- * This will start the process to teleport the funds to the corresponding Moonbeam address,
- * once the pix payment has arrived, and the reference and amount matches.
+ * Existing KYC level 1 user can request KYC level 2.
+ * This endpoint call brla and fetch the upload URLs for the documents.
  *
- * @returns void?.
+ * @returns Returns 200 if the documents were received successfully, and the corresponding URLs. 
  *
- * @throws 400 - If taxId is invalid. No Pay In could have been generated.
- * @throws 500 - For any server-side errors during processing
+ * @throws 400 - User does not exist, or is not yet KYC level 1 verified.
+ * @throws 500 - For any server-side errors during processing.
  */
-export const triggerPayIn = async (
-  req: Request<unknown, unknown, BrlaEndpoints.TriggerPayInRequest>,
-  res: Response<BrlaEndpoints.TriggerPayInResponse>,
+export const startKYC2 = async (
+  req: Request<unknown, unknown, BrlaEndpoints.StartKYC2Request>,
+  res: Response< BrlaEndpoints.StartKYC2Response | BrlaEndpoints.BrlaErrorResponse>,
 ): Promise<void> => {
   try {
-    const { taxId, amount, receiverAddress } = req.body;
+
+    const { taxId, documentType } = req.body;
 
     const brlaApiService = BrlaApiService.getInstance();
     const subaccount = await brlaApiService.getSubaccount(taxId);
+
     if (!subaccount) {
-      res.status(400).json({ error: 'taxId invalid' });
+      res.status(404).json({ error: 'Subaccount not found' });
+      return;
+    }
+
+    if (subaccount.kyc.level < 1) {
```

Comment:
Maybe it's better to explicitly check if it's `1`. We don't want to do it for any other level. If the account is at `0` or already at `2` it should fail here. 

```suggestion
    if (subaccount.kyc.level !== 1) {
```

Reply:
Correct.